jazzmin/templates/admin/base_site.html

```diff
@@ -3,65 +3,16 @@
 
-{% block title %}{{ title }} | {{ jazzmin_settings.site_title }}{% endblock %}
+{% block title %}
+    {{ block.super }}
+{% endblock %}
 
 {% block page_content %}
-    <div class="content-header">
-        <div class="container-fluid">
-            <div class="row mb-2">
-                <div class="col-auto col-sm-3">
-                    <h1 class="m-0 text-dark">{% block content_title %}{% endblock %}</h1>
-                </div>
-                <div class="col-auto col-sm-9">
-                    {% block breadcrumbs %}{% endblock %}
-                </div>
-            </div>
-        </div>
-    </div>
-
-    <div class="content">
-        <div class="container-fluid">
-            <section id="content" class="content">
-                {% block messages %}
-                    {% for message in messages %}
-                        {% if message.tags == 'success' %}
-                            <div class="alert alert-success alert-dismissible">
-                                <button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button>
-                                <i class="icon fa fa-check"></i>{{ message|capfirst }}
-                            </div>
-                        {% elif message.tags == 'error' %}
-                            <div class="alert alert-danger alert-dismissible">
-                                <button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button>
-                                <i class="icon fa fa-ban"></i>{{ message|capfirst }}
-                            </div>
-                        {% elif message.tags == 'warning' %}
-                            <div class="alert alert-warning alert-dismissible">
-                                <button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button>
-                                <i class="icon fa fa-exclamation-triangle"></i>{{ message|capfirst }}
-                            </div>
-                        {% elif message.tags == 'info' %}
-                            <div class="alert alert-info alert-dismissible">
-                                <button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button>
-                                <i class="icon fa fa-info"></i>{{ message|capfirst }}
-                            </div>
-                        {% endif %}
-                    {% endfor %}
-                {% endblock messages %}
-                <div class="row">
-                    {% block content %} {% endblock %}
-                </div>
-            </section>
-        </div>
-    </div>
-
+{{ block.super }}
 {% endblock %}
 
 {% block footer %}
-    <div class="float-right d-none d-sm-inline">
-        <b>Jazzmin Version</b> {% get_jazzmin_version %}
-    </div>
-
-    <strong>Copyright &copy; {% now 'Y' %} {{ jazzmin_settings.copyright }}.</strong> All rights reserved.
+    {{ block.super }}
 {% endblock %}
```

Comment:
If you are only going to leave `{{ block.super }}` in your `{% block %}`, you could just remove it all

Reply:
I totally agree.. left there for demonstrating purposes not more. It's cleaner to have just the extend line

Reply:
Sounds good to me 🙂